questionnaire-ko.json

```diff
@@ -1,434 +1,434 @@
+            	"3.2.1"
+              ]
+            },
+            "3.c.vi": {
+              "correctAnswer": "네",
+              "question": "     공급 대상 소프트웨어 내에서 상호 작용하는 다른 컴포넌트와 호환되지 않는 라이선스 하의 오픈소스 또는 기타 소프트웨어를 포함;",
+              "number": "3.c.vi",
+              "type": "YES_NO",
+              "specReference": [
                 "3.2",
-            	"3.2.1"
-              ]
-            }
-          }
-        }
-      ]
-    },
-    {
-      "name": "Part 4",
-      "title": "4. Compliance artifact creation and delivery",
-      "questions": [
-        {
-          "correctAnswer": "Yes",
-          "question": "Do you have a documented procedure describing the process for ensuring the Compliance Artifacts are distributed with Supplied Software as required by the Identified Licenses?",
-          "number": "4.a",
-          "type": "YES_NO",
-          "specReference": [
-            "4.1",
-            "4.1.1"
-          ]
-        },
-        {
-          "correctAnswer": "Yes",
-          "question": "Do you have a documented procedure for archiving copies of Compliance Artifacts for the Supplied Software?",
-          "number": "4.b",
-          "type": "YES_NO",
+            	"3.2.1"
+              ]
+            }
+          }
+        }
+      ]
+    },
+    {
+      "name": "Yes",
+      "title": "4. 3.c.vi",
+      "questions": [
+        {
+          "correctAnswer": "네",
+          "question": "Part 4",
+          "number": "4.a",
+          "type": "YES_NO",
           "specReference": [
-            "4.1",
-            "4.1.2"
-          ]
-        },
-        {
-          "correctAnswer": "Yes",
-          "question": "Are the Compliance Artifacts archived at least as long as the Supplied Software is offered and as required by the Identified Licenses?",
-          "number": "4.c",
-          "type": "YES_NO",
+            "4.1",
+            "4.1.1"
+          ]
+        },
+        {
+          "correctAnswer": "네",
+          "question": "식별된 라이선스에서 요구하는대로 컴플라이언스 결과물을 공급 대상 소프트웨어와 함께 배포하기 위한 프로세스를 설명하는 문서화된 절차가 있습니까?",
+          "number": "4.b",
+          "type": "YES_NO",
           "specReference": [
-            "4.1",
-            "4.1.2"
-          ]
-        }
-      ]
-    },
-    {
-      "name": "Part 5",
-      "title": "5. Understanding open source community engagements",
-      "questions": [
-        {
-          "correctAnswer": "Yes",
-          "question": "Do you have a policy for contribution to open source projects on behalf of the organization?",
-          "number": "5.a",
-          "type": "YES_NO",
+            "4.1",
+            "4.1.2"
+          ]
+        },
+        {
+          "correctAnswer": "네",
+          "question": "공급 대상 소프트웨어의 컴플라이언스 결과물 사본을 보관합니까?",
+          "number": "4.c",
+          "type": "YES_NO",
           "specReference": [
+            "4.1",
+            "4.1.2"
+          ]
+        }
+      ]
+    },
+    {
+      "name": "Yes",

```

Comment:
This is a bit messed up - see the following - looks like something got out of alignment during translation

Reply:
Indeed. I just fixed the alignment. I should have noticed that before. Sorry. PR later.